modules/_node-scripts/tsconfig/visitOutdatedTsconfigFiles.mjs

```diff
@@ -34,7 +34,12 @@ export default async function visitOutdatedTsconfigFiles(visitorFunction) {
  * {
  *	 '@liferay/frontend-js-react-web': {
  *		dir: 'modules/apps/frontend-js/frontend-js-react-web',
- *		path: 'src/main/resources/META-INF/resources/js/index.ts'
+ *		path: {
+ *			main: 'src/main/resources/META-INF/resources/js/index.ts',
+ *			submodules: {
+ *				foo: 'src/main/resources/META-INF/resources/js/foo.ts',
+ *			}
+ *		}
```

Comment:
nit: these paths should start with `./`

Reply:
I don't think so because they were like that before. I think we split them by `/` and rejoin them so if we added a `./` that would fail.